test/opset_pass/reverse_opset1upgrade_pass.cpp

```diff
@@ -0,0 +1,52 @@
+//*****************************************************************************
```

Comment:
We may want to put upgrade and downgrade pass tests in the same file. I think naming this `reverse_opset_pass.cpp` should be sufficient.